src/ragapp/admin-ui/sections/config/providers/ollama.tsx

```diff
@@ -109,40 +113,24 @@ export const OllamaForm = ({
           Could not fetch Ollama models. Make sure the Ollama base URL is
           accessible with RAGapp.
         </FormMessage>
-      ) : getLLMModels(models ?? []).length === 0 ? (
-        <FormMessage>
-          There is no LLM model available using Ollama. <br />
-          Please pull a Ollama LLM model from &nbsp;
-          <a href="https://ollama.com/library" target="_blank" rel="noreferrer">
-            https://ollama.com/library
-          </a>
-        </FormMessage>
-      ) : getEmbeddingModels(models ?? []).length == 0 ? (
+      ) : (
         <>
           <ModelForm
             form={form}
+            title="LLM Model"
+            description="Select an LLM model to use for language generation."
             defaultValue={defaultValues.model}
             supportedModels={getLLMModels(models ?? [])}
           />
-          <FormMessage>
```

Comment:
re-add the embedding model check, but change text to:
```
One of the embedding models <i>${embeddingModels.join(",")}</i> is required. Please pull an Ollama embedding model (e.g. 'nomic-embed-text') from &nbsp;
          <a href="https://ollama.com/library" target="_blank" rel="noreferrer">
            https://ollama.com/library
          </a>
```
```

Reply:
I've added the check for embedding, meanwhile I kept the LLM form if embedding model is missing. It seems to be working as expected.

![image](https://github.com/user-attachments/assets/37deaec4-c9d2-4055-8997-2c9900970847)